helper/testhelpers/testhelpers.go

```diff
@@ -33,7 +33,9 @@ const (
 	GenerateRecovery
 )
 
-// Generates a root token on the target cluster.
+const TOTPMFAWaitPeriod = 5
```

Comment:
It seems to me like it would be more obvious to the reader of a test using SetupLoginMFATOTP that this was the wait period it configured, if instead of this constant, you added an argument for wait period to SetupLoginMFATOTP.